### PR TITLE
refactor Account/Price.php - use look up table and change if conditio…

### PR DESCRIPTION
**Pull request by ualiyou**
…n syntax

### CODE REVIEW

File: src/Account/Price.php [hunk 1]
```diff
@@ -69,76 +61,71 @@ public function getPriceForNetwork($networkCode)
...
-
-                if (isset($n['network'])) {
-                    $n['networkName'] = $n['network'];
-                    unset($n['network']);
-                }
-
-                $network = new Network($n['networkCode'], $n['networkName']);
-                $network->jsonUnserialize($n);
-                $networks[$network->getCode()] = $network;
+        if (!isset($json['networks'])) {
```
Comment (dragonmantank) [inline src/Account/Price.php:89]: While the short-circuit return is nice, functionally it won't help much. We're replacing one `if` evaluation with another, so I don't see much gain for doing this. We're also losing the fact that there are empty networks, which the previous version would have recorded.

Comment (ualiyou) [reply src/Account/Price.php:89]: Yes it's my phpstorm config for reformat codes and i change config now,
Can I help refactor code in the process?

Comment (dragonmantank) [reply src/Account/Price.php:89]: Yes, helping refactor the code is always appreciated!

We just want to make sure that our code follows a similar process, and my notes above were more how we can refactor this in a more complete way, but any help from the community is great.

**Comment by dragonmantank** [comment]
Closing due to inactivity. If you'd like to work on this feel free to push an update with the changes requested above and and re-open the PR, we'll see what we can do!

**Pull request closed without merging**